PowershellTools.ProcessManager.Client/PowershellTools.ProcessManager.Client.csproj

```diff
@@ -0,0 +1,101 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I'm not sure whether we should have this wcf client factory as a standalone project as now, or it may just good enough with a separate folder("servicemanagement") under powershelltools project. The major reasons I m inclined to be settled as a folder in powershelltools project is that this client factory for wcf is pretty much working as a provider, which wont grow much once it is done in the first place. Also for any wcf duplex channel, we always want to pass some sort of context into channel, which apparently can be easier done/extend within one project.